Fix error due to conversion from integer constant to pointer.

Signed-off-by: Diego Nehab (Cartesi) <diego@cartesi.io>

diff -Naur mysql-5.1.73/client/mysql.cc mysql-5.1.73.patched/client/mysql.cc
--- mysql-5.1.73/client/mysql.cc	2013-11-04 18:52:27.000000000 +0000
+++ mysql-5.1.73.patched/client/mysql.cc	2018-10-05 21:27:47.468682913 +0000
@@ -2684,7 +2684,7 @@
         mysql_free_result(fields);
         break;
       }
-      field_names[i][num_fields*2]= '\0';
+      field_names[i][num_fields*2]= NULL;
       j=0;
       while ((sql_field=mysql_fetch_field(fields)))
       {
diff -Naur mysql-5.1.73/server-tools/instance-manager/instance_map.cc mysql-5.1.73.patched/server-tools/instance-manager/instance_map.cc
--- mysql-5.1.73/server-tools/instance-manager/instance_map.cc	2013-11-04 18:52:27.000000000 +0000
+++ mysql-5.1.73.patched/server-tools/instance-manager/instance_map.cc	2018-10-05 21:28:04.492927966 +0000
@@ -526,12 +526,12 @@
              Options::Main::config_file);
 
     argv_options[1]= defaults_file_arg;
-    argv_options[2]= '\0';
+    argv_options[2]= NULL;
 
     argc= 2;
   }
   else
-    argv_options[1]= '\0';
+    argv_options[1]= NULL;
 
   /*
     If the routine failed, we'll simply fallback to defaults in
